juniper_official/oam/ethernet-oam-lfm.rule

```diff
@@ -27,20 +27,34 @@ healthbot {
                 }
                 type string;
                 description "Discovery state for a given interface";
+                enumerate Passive Wait as 0;
+                enumerate Send Any as 1;
```

Comment:
Are you sure that enum names will work without double quotes?

I tried to load into terminal and getting below error.

         terminal:24:(38) syntax error: Wait
    'enumerate Passive Wait as 0;'
      syntax error